packages/api-v4/src/cloudpulse/types.ts

```diff
@@ -308,5 +306,14 @@ export type NotificationChannel =
   | NotificationChannelPagerDuty;
 
 export interface EditAlertDefinitionPayload {
-  entity_ids: string[];
+  entity_ids?: string[];
+  status?: AlertStatusType;
 }
+
+export interface EditAlertDefinitionPayloadWithServiceTypeAndAlertId
```

Comment:
```suggestion
export interface EditAlertPayloadWithService
```

lets rename it to much shorter version